spot-contracts/contracts/strategies/BasicFeeStrategy.sol

```diff
@@ -103,21 +119,46 @@ contract BasicFeeStrategy is IFeeStrategy, OwnableUpgradeable {
         emit UpdatedRolloverPerc(rolloverFeePerc_);
     }
 
+    /// @notice Enables debasement when the fee reserve is depleted.
+    function enableDebasement() public onlyOwner {
```

Comment:
I think I'd change this to `allowDebasement(bool allow)` and combine into one function.